web-components/src/[sandbox]/sandbox.scss

```diff
@@ -1,11 +1,36 @@
+md-theme {
+  background-color: white;
+  color: var(--md-primary-text-color);
+
+  &.is-visual-rebrand {
+    background: url("/images/brand-visuals/backgrounds/operation-pop-background-light.png");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+  }
+}
+
+
+md-theme[darkTheme] {
+  background-color: black;
+  color: var(--md-primary-text-color);
+
+  &.is-visual-rebrand {
+    background: url("/images/brand-visuals/backgrounds/operation-pop-background-light.png");
```

Comment:
should this be the dark version?

Reply:
currently there is no dark mode image. it is WIP so light mode temporarily set for both